bedrock/mozorg/templates/mozorg/home/includes/fundraising/takeover.html

```diff
@@ -0,0 +1,43 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+ <article id="fundraising_takeover" class="fundraising-takeover hidden">
+   <div class="header-strip">
+     <p class="contained"><a href="{{ url('mozorg.home') }}" id="close_takeover" class="more" role="button">{{ _('Continue to mozilla.org') }}</a></p>
+   </div>
+   <section class="copy contained">
+     <header>
+       <h1>{{ high_res_img('home/takeover/wordmark.png', {'alt': 'Mozilla', 'class': 'moz-logo', 'width': '276', 'height': '93'}) }}</h1>
```

Comment:
Yeah, I was thinking about that but, as the `article` starts a new context it is technically fine but, I know not all screen readers works very well with that document outline so, probably safer to just have two `h2` elements